angular/src/directives/control-value-accessors/select-value-accessor.ts

```diff
@@ -24,7 +24,14 @@ export class SelectValueAccessor implements ControlValueAccessor {
   onChange: (value: any) => void;
   onTouched: () => void;
 
+  /**
+   * Whether onChange should be mutted (not be fired). Will be true only when writeValue was called, which
+   * means that value changed inside angular form (e.g. calling setValue on a control).
+   */
+  private muteOnChange: boolean;
```

Comment:
I would give it a value by default
```suggestion
  private muteOnChange = false;
```

Reply:
@manucorporat 👍

Reply:
Sorry i meant this change for all the accessors

Reply:
Ups, my fault. Done!